.env.development

```diff
@@ -6,3 +6,6 @@ GOOGLE_ANALYTICS_APPLY=UA-APPLY-XX
 GOOGLE_ANALYTICS_MANAGE=UA-MANAGE-XX
 GOOGLE_TAG_MANAGER_APPLY=GTM-APPLY-XX
 GOOGLE_TAG_MANAGER_MANAGE=GTM-MANAGE-XX
+BIG_QUERY_API_JSON_KEY={}
+BIG_QUERY_PROJECT_ID=apply-bigquery-dataset
```

Comment:
I guess this is just a placeholder, but it's not right is it? Will we add documentation for how to configure this? Can be a separate PR.

Reply:
I'll add some info on configuration for BQ in docs and see if we even need these placeholders, will merge this one and update in next PR.